test/unit/helpers/buyers/applications_helper_test.rb

```diff
@@ -7,7 +7,7 @@ class Buyers::ApplicationsHelperTest < ActionView::TestCase
     service_ids = []
     service_ids << FactoryBot.create(:service_contract, user_account: buyer).service.id
     service_ids << FactoryBot.create(:service_contract, user_account: buyer).service.id
-    assert_equal services_contracted(buyer), service_ids.to_json
+    assert_same_elements JSON.parse(services_contracted(buyer)), service_ids
```

Comment:
I know that it is not your fault but the order of the elements is not correct... it should be first the expected result and second the actual result.
```ruby
assert_same_elements service_ids, JSON.parse(services_contracted(buyer))
```
I know that it seems like it doesn't matter but when it fails the message is very confusing with the wrong order.

Reply:
Please check this: https://github.com/3scale/porta/pull/523 😄